src/lib/apis/maxim.ts

```
@@ -1,109 +1,224 @@
+
+		// Create axios instance with optimal configuration
+		this.axiosInstance = axios.create({
+			baseURL: baseUrl,
+			timeout: 30000, // 30 second timeout
+			headers: {
+				"User-Agent": "Maxim-SDK/1.0",
+				Accept: "application/json",
+				Connection: "keep-alive",
+			},
+			// Enable connection pooling and keep-alive
+			httpAgent: new (require("http").Agent)({
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)